chain/events/filter/index.go

```diff
@@ -55,8 +53,6 @@ var ddls = []string{
 
 	createTableEventsSeen,
 
-	createIndexEventEntryIndexedKey,
```

Comment:
Why do we need to remove the `createIndexEventEntryIndexedKey` AND `createIndexEventEntryCodecValue` indices > Where do they get in the way ?

Reply:
the sqlite index selection story gets worse, consider the case I outlined above with the secondary join and `AND ((ee2.value=X'DDF252AD1BE2C89B69C2B068FC378DAA952BA7F163C4A11628F55A4DF523B3EF' AND ee2.codec=85))`, if I run a query plan with and without this index:

```
QUERY PLAN
|--SEARCH ee2 USING INDEX event_entry_codec_value (codec=? AND value=?)
|--SEARCH event USING INTEGER PRIMARY KEY (rowid=?)
|--SEARCH event_entry USING INDEX event_entry_event_id (event_id=?)
`--USE TEMP B-TREE FOR ORDER BY
```

without:

```
QUERY PLAN
|--SEARCH event USING INDEX event_height (height>? AND height<?)
|--SEARCH ee2 USING INDEX event_entry_event_id (event_id=?)
|--SEARCH event_entry USING INDEX event_entry_event_id (event_id=?)
`--USE TEMP B-TREE FOR RIGHT PART OF ORDER BY
```

it reaches in to event_entry indexes (!!) to find the first index to narrow down the selection; it ends up being not-terrible because the `value` part of the index is relatively unique. But imagine a really popular smart contract, running thousands or millions of times over the life of this database, that's significantly worse than first narrowing down by epoch.